README.md

```diff
@@ -2,12 +2,16 @@
 
 This repo is the canonical source for Operators that appear in [OperatorHub.io](https://operatorhub.io).
 
+## Before contribution
+
+All Operators on [OperatorHub.io](https://operatorhub.io) need to function on vanilla Kubernetes. If you mandate a certain Kubernetes version you express this in the metadata (see below).
+
 ## Adding your Operator
 
-To add your operator to this repo, you will need to submit a PR with your Operator resources in a new directory named after your Operator within the `community-operators/` directory:
+To add your operator to this repo, you will need to submit a PR with your Operator resources in a new directory named after your Operator within the `upstream-community-operators/` directory:
```

Comment:
What's the difference between:

- community-operators
- upstream-community-operators
- redhat-operators
- certified-operators

...is something I want to be able to learn by reading this file. Would you add some brief treatment of that in this PR?

Reply:
please have a look